modules/statistics/php/stats_demographic.class.inc

```diff
@@ -98,7 +99,7 @@ class Stats_Demographic extends \NDB_Form
         $tpl_data['DropdownName']     = $dropdown_name;
         $tpl_data['DropdownOptions']  = $dropdown_opt;
         $tpl_data['DropdownSelected'] = $dropdown_selected;
-        $tpl_data['Centers']          = $centres;
+        $tpl_data['Centers']          = $centers ?? array();
```

Comment:
Same here, short syntax.

Reply:
I see there are a lot of old `array()` style declarations below also. Would be nice to have these updated but it's not essential. I won't add more comments for each instance.